llm/cpp/llm.cpp

```diff
@@ -2,41 +2,24 @@
 // SPDX-License-Identifier: Apache-2.0
 
 #include <openvino/openvino.hpp>
+#include <utils.hpp>
```

Comment:
File name is actually ambiguous, not sure if this is not error prone. 
I understand that it is tokenizers design though...
cc: @apaniukov

Reply:
Add necessary definitions to `CMakeLists.txt` to include it as `tokenizer/utils.hpp` as `utils.hpp` is in that directory.

Reply:
Added